test/e2e/performanceprofile/functests/1_performance/cpu_management.go

```diff
@@ -671,6 +671,7 @@ func checkPodHTSiblings(testpod *corev1.Pod) bool {
 	Expect(err).ToNot(
 		HaveOccurred(), "Unable to cpuset.Parse pod allocated cpu set from output %s", fmt.Sprint(output))
 	By(fmt.Sprintf("Test pod CPU list: %s", podcpus.String()))
+	testlog.Infof("Test pod CPU list: %s", podcpus.String())
```

Comment:
I agree. Do we want to add or replace? I'm slightly in favor of replacing. Your call.

Reply:
nevermind, you actually delete it in the next patch. Good!